week1/week1.py

```diff
@@ -0,0 +1,58 @@
+##########Global Variables##########
+
+###########Primitive Data Types##########
+#python isn't a typed language like C++ or Java
```

Comment:
Python is a strongly typed language...

Python is a dynamically typed language, unlike C++ or java. (or)
Python isn't a statically typed language, like C++ or java.